src/main/java/com/exactpro/th2/act/ActMain.java

```diff
@@ -38,6 +39,8 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ActMain.class);
     private static final String OE_ATTRIBUTE_NAME = "oe";
+    private static final MetricMonitor LIVENESS_MONITOR = registerLiveness("act_liveness");
+    private static final MetricMonitor READINESS_MONITOR = registerReadiness("act_readiness");
```

Comment:
Shouldn't we use a common monitor instead? The act does not have extra metrics

`LIVENESS_MONITOR` and `READINESS_MONITOR` from `com.exactpro.th2.common.metrics`

Reply:
Actually, preregistered monitors are the same as newly created ones, the only difference is the name. 
If it's better to use existing ones - I will correct the code.